### DISCUSSION TITLE
Not all devices in app are appearing

### DISCUSSION THREAD

**Discussion by mouse1974** [milo526/homebridge-tuya-web#323] [Q&A]
Hi there I have around 25 devices on the Tuya App that are showing up on the TUYA development site. This plugin work amazing for most of these but for some they are not being seen within Homebridge and I am at a loss to work out why. They are the same devices and all with the same firmware?? Are you able to help in any way?

**Comment by milo526** [comment]
Please post the result of [the steps under "How to check whether the api this library uses can control your device".](https://github.com/milo526/homebridge-tuya-web#how-to-check-whether-the-api-this-library-uses-can-control-your-device)

Without this it is hard for me to help you with regards to this specific issue.

**Reply by many7695** [reply]
I am having the same issue with it not finding moon light projector, it’s available to use on the smart life app and on google and Alexa
